learningresources/tests/test_api.py

```diff
@@ -157,36 +152,6 @@ def test_invalid_create_repo(self):
         self.assertIn("description", ex.exception.args[0])
 
 
-class TestStaticAsset(LoreTestCase):
```

Comment:
It is the only unit level test for that method, and it isn't causing errors on master...It is generally never a good idea to remove a test, and multiple coverage is important, generally you should have tests for each individual function, and then a few integration tests, not just integration tests just for coverage.